examples/generate-charity-check-pdf/main.bal

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2024, WSO2 LLC. (http://www.wso2.org).
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+import ballerina/os;
+import ballerinax/candid.charitycheckpdf;
+
+configurable string subscriptionKey = os:getEnv("SUBSCRIPTION_KEY");
```

Comment:
Since we have already mentioned in the Readme that the user need to configure this parameter in the `Config.toml` I think we do not need to check for environment variable.
```suggestion
configurable string subscriptionKey = ?;
```